src/global/translations/en.json

```diff
@@ -13,6 +13,9 @@
       },
       "create": {
         "succeed": "Created Nation {{nationName}}"
+      },
+      "leave": {
+        "succeed": "Left nation: {{nationName}}"
```

Comment:
Do we need the colon here?

Reply:
I've just followed the text from joined nation, so I'm not sure.

Reply:
Does it matter?

Reply:
Or I guess it does since @SeniorMobileDeveloper asked for it. But why exactly is it important?